vite/msdfGenerator.js

```diff
@@ -158,6 +163,44 @@ const copyDir = (src, dest) => {
     const srcPath = path.join(src, entry.name)
     const destPath = path.join(dest, entry.name)
 
+    // Skip copying charset.txt file into dist directory
+    if (srcPath.includes('.charset.txt')) continue
+
     entry.isDirectory() ? copyDir(srcPath, destPath) : fs.copyFileSync(srcPath, destPath) // Copy files
   }
 }
+
+const createCharsetFile = (charsetDir, ConfigDir, fontName) => {
+  const charsetFilePath = path.join(charsetDir, `${fontName}.charset.txt`)
+
+  // Check charset file already exists, if exists, return file path
+  if (fs.existsSync(charsetFilePath)) {
+    return charsetFilePath
+  }
+
+  const configFilePath = path.join(ConfigDir, `${fontName}.config.json`)
+
+  // Check if font config.json is exists
+  if (fs.existsSync(configFilePath)) {
+    if (!fs.existsSync(charsetDir)) {
+      fs.mkdirSync(charsetDir, { recursive: true })
+    }
+    const configData = JSON.parse(fs.readFileSync(configFilePath, 'utf8'))
+    let charsetContent = configData.charset
+    const presets = configData.preset
+    for (let i = 0; i < presets.length; i++) {
+      charsetContent = charsetContent + presets[i]
+    }
+
+    // write charset content into .txt file
+    try {
+      fs.writeFileSync(charsetFilePath, charsetContent)
```

Comment:
is the .txt file added to the public/assets folder? or the 'hidden' folder in node_modules?

Reply:
it will be part of ```.tmp-msdf-fonts-v2``` directory and ```.txt``` file won't be copied to ```dist``` directory

Reply:
This comment no longer required as charset text file generation completely removed from vite plugin.